src/components/SingletonEditor.tsx

```diff
@@ -95,7 +95,7 @@ export const useSingletonEditor = (
 
       if (currentUpdateCounter >= savedCounter.current) {
         savedCounter.current = currentUpdateCounter;
-        onSave(content as Content);
+        onSave(content);
```

Comment:
@Cromeror I am happy about this change, now `Content` means only _the content_, without the `campaignName`, and it is what the SingletonEditor deals with.

Reply:
![image](https://user-images.githubusercontent.com/1157864/201187111-afec7352-13b3-41dd-b5be-460a61c4000b.png)